storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -1,84 +1,160 @@
+## Tekstgrootte
 
-### Tekstniveaus
+De grootte van de tekst is afhankelijk van de breedte van het venster.
+Hiermee zorgen we ervoor dat tekst op schermen van alle afmetingen prettig leesbaar is.
 
-We hebben 8 tekstniveaus gedefinieerd.
+Er zijn 8 niveaus van tekstgrootte en bijbehorende regelhoogte.
 Alle typografische componenten maken er gebruik van.
+Dit overzicht toont de maximale tekstgrootte voor elk niveau.
 
-Een tekstniveau definieert de tekstgrootte op de minimale vensterbreedte (320 pixels) waaronder de tekst niet kleiner wordt, de tekstgrootte op de maximale vensterbreedte (1440 pixels) waarboven de tekst niet groter wordt, de tekst op het breekpunt (854 pixels) en de regelhoogte.
-De tekstgrootte op het breekpunt is de minimale waarde + 2/3 van het verschil tussen de minimale en maximale waarde.
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[80, 56, 40, 32, 28, 25, 22, 18]}
+  fontWeight={400}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+/>
+
+Elk tekstniveau heeft een minimale tekstgrootte (bij een vensterbreedte van 320 pixels) en een maximale (bij 1440 pixels).
+Tussen beide stijgt de tekst linear, met een knik op een extra breakpoint van 854 pixels.
+Eerst stijgt de grootte sneller, daarna langzamer.
+
+Op dit moment reageert de typografie nog op 1 breakpoint bij een vensterbreedte van 854 pixels.
+Nu het grid doorontwikkeld is, stemmen we de typografie daar weer op af.
+Dit aparte breakpoint komt dus snel te vervallen.
 
 In de tabel hieronder vind je deze waardes in pixels:
 
-| niveau                     |   0 |   1 |   2 |   3 |   4 |   5 |   6 |   7 |
+| Niveau                     |   0 |   1 |   2 |   3 |   4 |   5 |   6 |   7 |
 | :------------------------- | --: | --: | --: | --: | --: | --: | --: | --: |
 | minimale tekstgrootte      |  32 |  32 |  28 |  24 |  20 |  22 |  18 |  16 |
 | tekstgrootte op breakpoint |  64 |  48 |  36 |  29 |  23 |  26 |  21 |  17 |
 | maximale tekstgrootte      |  80 |  56 |  40 |  32 |  25 |  28 |  22 |  18 |
 | relatieve regelhoogte      | 1.1 | 1.2 | 1.3 | 1.4 | 1.4 | 1.5 | 1.6 | 1.6 |
 
+Tekst op niveau 5 is groter dan op niveau 4.
+Dat is omdat titel niveau 4 kleinere letters heeft dan paragrafen met grote letters.
+Het alternatief om Heading 4 tekstniveau 5 te geven leek ons nog verwarrender.
+
 Voorbeelden van componenten en hun tekstniveaus:
 
-| level | componenten                          |
-| ----: | :----------------------------------- |
-|     0 | Page heading                         |
-|     1 | Heading 1                            |
-|     2 | Heading 2                            |
-|     3 | Heading 3, Blockquote                |
-|     4 | Heading 4, Top Task Link             |
-|     5 | Paragraph large                      |
-|     6 | Paragraph, alle Links behalve inList |
-|     7 | Paragraph small, inList Link         |
+| Componenten                                          | Niveau |
```

Comment:
First 'Niveau' and then 'Componenten' was more readable imo, because we're talking about the text levels in this section.